polkadot-parachains/statemint-common/src/impls.rs

```diff
@@ -61,6 +62,21 @@ where
 	}
 }
 
+pub struct AssetsToBlockAuthor<R>(PhantomData<R>);
+impl<R> pallet_asset_tx_payment::HandleCredit<AccountIdOf<R>, pallet_assets::Pallet<R>> for AssetsToBlockAuthor<R>
+where
+	R: pallet_authorship::Config + pallet_assets::Config,
+	AccountIdOf<R>: From<polkadot_primitives::v1::AccountId> + Into<polkadot_primitives::v1::AccountId>,
+{
+	fn handle_credit(credit: CreditOf<AccountIdOf<R>, pallet_assets::Pallet<R>>) {
+		let author = pallet_authorship::Pallet::<R>::author();
+		// TODO: what to do in case paying the author fails (e.g. because `fee < min_balance`)
```

Comment:
Is this really a TODO?

Reply:
Not really. Just wanted to consider this again.
Will fix by having a more sophisticated (pot-based) fee dispersal later.